web-test-runner-it.config.js

```diff
@@ -1,13 +1,15 @@
 /* eslint-env node */
-const { puppeteerLauncher } = require('@web/test-runner-puppeteer');
 const { createIntegrationTestsConfig } = require('./wtr-utils.js');
 const devServerConfig = require('./web-dev-server.config.js');
+const { playwrightLauncher } = require('@web/test-runner-playwright');
 
 const unitTestsConfig = createIntegrationTestsConfig({
   browsers: [
-    puppeteerLauncher({
+    playwrightLauncher({
```

Comment:
Let's also update the config for snapshot tests accordingly for consistency. 
We can then drop the `@web/test-runner-puppeteer` dependency altogether.